### DISCUSSION TITLE
Welcome to UE4-TDD-CI_Testing Discussions!

### DISCUSSION THREAD

**Discussion by Floating-Island** [Floating-Island/UE4-TDD-CI_Testing#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by lilitttaa** [comment]
Thank you very much for your work, you've been a great help to me.

**Reply by Floating-Island** [reply]
Hi! 
I'm glad it was useful for you 😄 
Happy New Year! 🎆

**Reply by lilitttaa** [reply]
Happy New Year!!!  🎇
